backend/models/util.py

```diff
@@ -25,6 +25,9 @@ def parse_date(date_str):
     (y,m,d) = date_str.split('-')
     return date(int(y),int(m),int(d))
 
+def parse_unix_timestamp_to_datetime(timestamp, agency_tz):
```

Comment:
The function name is a bit misleading since the return value is a string rather than a datetime, and parse also implies that the parameter is a string. Maybe could just call it `unix_timestamp_to_datetime` and move the `.strtftime` call to the caller since this format is specific to the CSV download.

Reply:
+1, I like that idea.